_posts/2017-04-12-easy-test-deployments.md

```diff
@@ -0,0 +1,295 @@
+---
+layout: post
+title: Easily deploying test environments from CI builds with Travis, AWS and Slack
+author: tatut
+excerpt: Easy setup of test environments is important for feature branch based development. This post shows how to automatically deploy a new test environment with a simple chat interface.
```

Comment:
en tajunnut tästä excerptistä ollenkaan mistä on kyse tuossa chat interfacessa :) Kyllä se sieltä selvisi lopulta, mutta ehkä tämän voisi ilmaista jotenkin selkeämmin, jotain tällaista?
 
"This post shows how to automatically deploy builds to AWS with Travis CI and how developers can control the deployments with a a simple chat interface."